tmt/steps/execute/internal.py

```diff
@@ -91,6 +110,12 @@ def effective_pidfile_root() -> Path:
     {{ REMOTE_COMMAND }};
     _exit_code="$!";
     {{ exit() }};
+{% elif TTY %}
+    set -o pipefail;
+    {{ enter() }};
+    {{ REMOTE_COMMAND }} </dev/null 2>&1;
```

Comment:
Why is the `</dev/null` here when we want test to have tty available?

Reply:
ah this last thing

Reply:
resolved